### ISSUE TITLE
Dockerfile can't be build properly 

### ISSUE THREAD

**Issue by filipe96** [kalessil/phpinspectionsea#1175]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Docker build failed                          |
| Plugin         |not important for the bug|
| Language level | not important for the bug                       |

Current behaviour (description/screenshot:)
~~~~
 ---> Running in 790c6a73038d
The command '/bin/sh -c cd root; mkdir -p .PhpStorm${PHPSTORMVERSION}/config .PhpStorm${PHPSTORMVERSION}/system /var/ci/project && wget --quiet --no-verbose https://download.jetbrains.com/webide/PhpStorm-${PHPSTORMDOWNLOAD}.tar.gz && tar -xf PhpStorm-*.tar.gz && cd PhpStorm-*/plugins && ls | grep -Evi "^(css|php)?$" | xargs rm -rf && wget --quiet --no-verbose https://download.plugins.jetbrains.com/7622/38887/PhpInspectionsEA.jar && cd - && wget --quiet --no-verbose https://getcomposer.org/composer.phar && mv composer.phar /usr/local/bin/composer && chmod +x /usr/local/bin/composer && composer require --optimize-autoloader kalessil/phpstorm-inspect' returned a non-zero code: 4
~~~~

--

Expected behaviour
~~~~
Successfully built d34815cad76c
Successfully tagged phpinspectionsea:latest
~~~~

--

Environment details
---
Information from Help -> About dialog (contains IDE and it's version details).

Notes
---
- That my first time I contribute to open Source :)
- I have the solution already ( I only must figure out how to contribute it)

**Comment by kalessil** [comment]
@filipe96, I can accept even patch-file if it makes contribution easier for you =)

Regular workflow is to fork the repository on GitHub, make changes in your form and create a Pull Request on GitHub.

**Comment by kalessil** [comment]
Merged #1182
